collector/arp_linux.go

```diff
@@ -75,30 +96,38 @@ func getTotalArpEntries(deviceEntries []procfs.ARPEntry) map[string]uint32 {
 	return entries
 }
 
-func getTotalArpEntriesRTNL() (map[string]uint32, error) {
+func getArpEntriesNTRL() (map[string]uint32, map[string]map[string]int, error) {
```

Comment:
Letters transposed in function name - RTNL (routing netlink) -> NTRL (?)

Reply:
Hi Daniel, fixed that.